gene2phenotype_project/gene2phenotype_app/serializers/user.py

```diff
@@ -1,7 +1,13 @@
 from rest_framework import serializers
 from django.core.exceptions import ValidationError
+from django.utils.http import urlsafe_base64_decode, urlsafe_base64_encode
+from ..utils import CustomMail
 from ..models import User, UserPanel
```

Comment:
Can you move these two lines to the end (after line 11)?
This import is not the same as the other imports (from packages), separating them is easier to understand what we are importing from gene2phenotype_app.